pyartcd/pyartcd/pipelines/prepare_release_konflux.py

```diff
@@ -96,6 +100,14 @@ def __init__(
             f'--data-path={self.build_repo_pull_url}',
         ]
 
+        self._doozer_base_command = [
+            'doozer',
```

Comment:
why not pass in assembly param?

Reply:
@vfreex maybe you have thoughts here?

Reply:
its due to this line https://github.com/openshift-eng/art-tools/blob/main/doozer/doozerlib/cli/images_konflux.py#L369, maybe the no need to check the `group` key